main.c:125:6: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
--- main.c.orig	2000-06-22 17:58:07.000000000 -0500
+++ main.c	2021-02-14 04:24:17.000000000 -0600
@@ -24,6 +24,7 @@
 
 #include <time.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include <signal.h>
 #include <fcntl.h>
